projects/valtimo/user-interface/src/lib/components/input/input.component.ts

```diff
@@ -93,6 +93,14 @@ export class InputComponent implements OnInit, OnChanges, OnDestroy {
     this.clearSubscription?.unsubscribe();
   }
 
+  onKeydown(event: KeyboardEvent): void {
+    if (event.code !== 'Enter') {
```

Comment:
Not sure why you're inverting the comparison, doesn't this:
```
if (event.code === 'Enter') {
    event.preventDefault();
}
```
accomplish the same?

Reply:
As a general best practice, using early returns reduces method nesting and improves code readability